requirements-igwn.txt

```diff
@@ -1,4 +1,5 @@
 # For LDG service access
+ligo-proxy-utils
```

Comment:
What is this needed for?

Reply:
@spxiwh That's a question for you all actually. If it's not needed we can remove it. However, it was hard coded in the workflow whereas if it is needed it should be in the requirements rather than installed by hand.